File: app/src/main/java/com/artamonov/millionplanets/utils/RandomUtils.java
package com.artamonov.millionplanets.utils;

import java.util.Random;

public class RandomUtils {
    private static final String ALLOWED_CHARACTERS = "0123456789qwertyuiopasdfghjklzxcvbnm";

    public static String getRandomDebrisName(final int sizeOfRandomString) {
        final Random random = new Random();
        final StringBuilder sb = new StringBuilder(sizeOfRandomString);
        for (int i = 0; i < sizeOfRandomString; ++i)
            sb.append(ALLOWED_CHARACTERS.charAt(random.nextInt(ALLOWED_CHARACTERS.length())));
        return sb.toString();
    }

    public static int getRandomCoordinate() {
        Random r = new Random();
        return r.nextInt(600 - 500) + 500;
    }

    public static int getRandomDebrisIron() {
        Random r = new Random();
        int min = 100;
        int max = 300;
        return r.nextInt(max - min + 1) + min;
    }
}

File: app/src/main/java/com/artamonov/millionplanets/utils/RandomUtils.kt
package com.artamonov.millionplanets.utils

import java.util.Random

object RandomUtils {

    private const val ALLOWED_CHARACTERS = "0123456789qwertyuiopasdfghjklzxcvbnm"
    fun getRandomDebrisName(sizeOfRandomString: Int): String {
        val random = Random()
        val sb = StringBuilder(sizeOfRandomString)
        for (i in 0 until sizeOfRandomString) sb.append(ALLOWED_CHARACTERS[random.nextInt(ALLOWED_CHARACTERS.length)])
        return sb.toString()
    }

    val randomCoordinate: Int
        get() {
            val r = Random()
            return r.nextInt(600 - 500) + 500
        }

    val randomDebrisIron: Int
        get() {
            val r = Random()
            val min = 100
            val max = 300
            return r.nextInt(max - min + 1) + min
        }
}